config/locales/interview.yml

```diff
@@ -26,10 +26,12 @@ en:
         page_title: Cancel Interview
         title: Reason for cancelling interview
         success: Interview cancelled
+        cancel: Cancel
       review_cancel:
         page_title: Cancel interview
         title: Check and send interview cancellation
         cancellation_reason: Reason for cancelling interview
+        cancel: Cancel
```

Comment:
Can we move this in `en.yml` as there are cancel components across the entire app and this will also enble reuse. And update any related I18n calls `t('.cancel')` -> `t('cancel')`